worlds/cvcotm/presets.py

```diff
@@ -0,0 +1,159 @@
+    "completion_goal":               CompletionGoal.option_dracula,
+}
+
+randomania_2023_options = {
+    "progression_balancing":         ProgressionBalancing.default,
+    "accessibility":                 Accessibility.option_items,
+    "ignore_cleansing":              IgnoreCleansing.option_false,
+    "auto_run":                      AutoRun.option_false,
+    "dss_patch":                     DSSPatch.option_true,
+    "always_allow_speed_dash":       AlwaysAllowSpeedDash.option_true,
+    "break_iron_maidens":            BreakIronMaidens.option_false,
+    "required_last_keys":            3,
+    "available_last_keys":           5,
+    "buff_ranged_familiars":         BuffRangedFamiliars.option_true,
+    "buff_sub_weapons":              BuffSubWeapons.option_true,
+    "buff_shooter_strength":         BuffShooterStrength.option_false,
+    "item_drop_randomization":       ItemDropRandomization.option_normal,
+    "halve_dss_cards_placed":        HalveDSSCardsPlaced.option_false,
+    "countdown":                     Countdown.option_majors,
+    "sub_weapon_shuffle":            SubWeaponShuffle.option_true,
+    "disable_battle_arena_mp_drain": DisableBattleArenaMPDrain.option_false,
+    "require_all_bosses":            RequireAllBosses.option_false,
+    "early_double":                  EarlyDouble.option_false,
+    "death_link":                    DeathLink.option_off,
+    "completion_goal":               CompletionGoal.option_dracula,
+}
+
+competitive_all_bosses_options = {
+    "progression_balancing":         ProgressionBalancing.default,
+    "accessibility":                 Accessibility.option_items,
+    "ignore_cleansing":              IgnoreCleansing.option_false,
+    "auto_run":                      AutoRun.option_false,
+    "dss_patch":                     DSSPatch.option_true,
+    "always_allow_speed_dash":       AlwaysAllowSpeedDash.option_true,
+    "break_iron_maidens":            BreakIronMaidens.option_false,
+    "required_last_keys":            8,
+    "available_last_keys":           8,
+    "buff_ranged_familiars":         BuffRangedFamiliars.option_true,
+    "buff_sub_weapons":              BuffSubWeapons.option_true,
+    "buff_shooter_strength":         BuffShooterStrength.option_false,
+    "item_drop_randomization":       ItemDropRandomization.option_hard,
+    "halve_dss_cards_placed":        HalveDSSCardsPlaced.option_true,
+    "countdown":                     Countdown.option_none,
+    "sub_weapon_shuffle":            SubWeaponShuffle.option_true,
+    "disable_battle_arena_mp_drain": DisableBattleArenaMPDrain.option_false,
+    "require_all_bosses":            RequireAllBosses.option_true,
+    "early_double":                  EarlyDouble.option_false,
+    "death_link":                    DeathLink.option_off,
+    "completion_goal":               CompletionGoal.option_dracula,
+}
+
+hardcore_mode_options = {
+    "progression_balancing":         0,
```

Comment:
I don't recommend having progression balancing being set to 0 under hardcore options. If someone brings hardcore to a sync and doesn't realize this, it could potentially ruin the sync due to early BK.

Reply:
I took a bit of inspiration from Stardew Valley's presets, but maybe it was a bit more than I should've, lol.